models/cam-fv/shell_scripts/cesm2_1/assimilate.csh.template

```diff
@@ -98,6 +91,15 @@ else if ($?LSB_HOSTS) then
 
 endif
 
+if ($?JOBNAME) then
```

Comment:
note for xmlquery (not critical for this review)
this bit (and above) is the machine specific stuff that does not use xmlquery

Reply:
Yes, that's the sort of variable I mentioned that comes directly from the job environment,
instead of from the CESM environment.
The COMP_ATM variable is not machine specific.  It describes the atmospheric component
that's specified in the CESM component set.